sqlite-android/src/main/java/io/requery/android/database/sqlite/SQLiteGlobal.java

```diff
@@ -97,20 +97,20 @@ public static String getDefaultSyncMode() {
      * Gets the database synchronization mode when in WAL mode.
      */
     public static String getWALSyncMode() {
-        return "FULL";
+        return "normal";
```

Comment:
Just for reference in case someone else is curious, this is the artifact upstream, these changes seem right on

https://sqlite.org/android/annotate?filename=sqlite3/src/main/java/org/sqlite/database/sqlite/SQLiteGlobal.java&checkin=253313a7ee8cc014